target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,13 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_usbdev "phone" "Phone" "${board}:white:phone" "1"
```

Comment:
Actually this is wrong. I did not want to control the phone LED at all here. I am currently misusing it as status LED as I could not figure out all pins.